resources/store/index.js

```diff
@@ -13,7 +16,9 @@ var Vue = require( 'vue' ),
 Vue.use( Vuex );
 
 // Set up API client
-api = new mw.Api();
+api = wikibase.api.getLocationAgnosticMwApi(
```

Comment:
The linting settings are going to need to be updated to handle implicit dependencies on global `wikibase` object unless there's a way to `require()` it. Right now Vuex is not included in our unit tests so we won't have to worry about mocking this stuff for now.